dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -86,10 +86,10 @@
             <Color x:Key="LayerOnAcrylicFillColorDefault">#09FFFFFF</Color>
             <Color x:Key="LayerOnAccentAcrylicFillColorDefault">#09FFFFFF</Color>
 
-            <Color x:Key="LayerOnMicaAltFillColorDefault">#733A3A3A</Color>
```

Comment:
There have been CBS packages released since this resource was introduce, which means there is a chance that a consumer of the CBS package is dependent on this named resource and changing the name would break them.  To fix this we can either look through the CBS consumers to ensure this wont break, or add the previous name to the DeprecatedResources dictionary.